src/prefect/agent/kubernetes/agent.py

```diff
@@ -562,6 +562,8 @@ def generate_job_spec(self, flow_run: GraphQLResult) -> dict:
         env.update(self.env_vars)
         if run_config.env:
             env.update(run_config.env)
+        if env.get("PREFECT__CLOUD__SEND_FLOW_RUN_LOGS", "").lower() != "false":
+            env["PREFECT__CLOUD__SEND_FLOW_RUN_LOGS"] = str(self.log_to_cloud).lower()
```

Comment:
I think what we want here is:
```suggestion
        # Allow logs to be disabled by flow run settings but not enabled
        if not self.log_to_cloud or "PREFECT__CLOUD__SEND_FLOW_RUN_LOGS" not in env:
           env["PREFECT__CLOUD__SEND_FLOW_RUN_LOGS"] = str(self.log_to_cloud).lower()
```

Reply:
Gotcha